tests/test_issues/test_linkml_issue_1126.py

```diff
@@ -0,0 +1,28 @@
+import logging
+import unittest
+from unittest import TestCase
+from linkml_runtime.utils.schemaview import SchemaView
+from linkml_runtime.linkml_model import ClassDefinition
+
+from tests.test_issues.environment import env
+
+
+class Issue998TestCase(TestCase):
```

Comment:
`class Issue998TestCase` or `class Issue1126TestCase`? Very trivial but just noticed it